templates/master/00-master/openstack/files/openstack-haproxy.yaml

```diff
@@ -99,7 +99,7 @@ contents:
         livenessProbe:
           initialDelaySeconds: 10
           httpGet:
-            path: /healthz
+            path: /readyz
```

Comment:
Isn't /readyz endpoint used usually for readiness check and /healthyz used for liveness check?
If that is that is the case, I think we should consider keeping /healthz here, as it's used for HAProxy pod Liveness check